tests/queries/0_stateless/03133_help_message_verbosity.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
```

Comment:
I am indifferent if we need a test (I personally would not have written one but I am good with a test).

In any case, what you (generally) want to do is to add a one-line summary of the test purpose. Imagine a future programmer reading this file in a few months from now. The file name could mean many things but not necessarily `clickhouse-client --help`. So the only option to understand what is actually tested is to read the test :-(

Therefore, let's add here something like `# Checks that "clickhouse-client/local --help" prints a brief summary of CLI arguments and "--help --verbose" prints all possible CLI arguments`

Reply:
That's a really good point, will add the comment, thanks